apps/condo/domains/ticket/utils/tables.utils.ts

```diff
@@ -67,4 +70,43 @@ export const getPropertyScopeFilter = () => {
             console.error(e)
         }
     }
-}
\ No newline at end of file
+}
+
+/**
+ * Getting a where query for an address in the search field.
+ * All punctuation marks and stop words are removed.
+ * Also remove short word (less 3 symbols)
+ *
+ * @example
+ *  search: 'ул. ленина д 1/1'
+ *  whereQuery: {
+ *      property: {
+ *          AND: [
+ *              { address_contains_i: 'ленина' },
+ *              { address_contains_i: '1/1' },
+ *          ],
+ *      },
+ *  }
+ */
+export const getFilterAddressForSearch = (): FilterType => {
+    return function getWhereQuery (search) {
+        if (!search || !isString(search)) return
+
+        const addresses = uniq(search
+            .replace(/[^\p{Alphabetic}\p{Decimal_Number}\s/]/igu, ' ')
```

Comment:
Moving this to const can be more efficient in terms of future fixes / refactoring